README.md

```diff
@@ -1 +1,142 @@
-# log-parser
+## Speedb-Log-Parser: A tool for parsing RocksDB and Speedb Log Files
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+Speedb's Log Parser is a tool that may be used to parse and process 
+Speedb and RocksDB log files.
+
+The tool extracts useful information from these logs and aids users in 
+gaining insights about their systems with ease. It is expected to be valuable tool
+for novices and experts alike.
+
+
+
+
+<!-- GETTING STARTED -->
+## Getting Started
+The tool runs on a single log file and generates one or more outputs:
+1. A short summary printed to the console
+2. A detailed summary in a JSON format
+3. The detailed JSON file printer to the console. This may be filtered 
```

Comment:
"a detailed JSON file printed to the console" ?

Reply:
Yes, @ofriedma proposed doing that. The detailed information in the json contains tons of information. Reading it on the console is not so practical or convenient. However, there are widely-available command line tools that are able to parse json (e.g., JQ) and allow proficient console users to easily filter the json on the console (only those would print the detailed information to the console anyway)

Reply:
also, just a spelling fix